test/levels_test.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env rspec
+
+require_relative 'test_helper'
+
+module Yast
+  class LevelsTester
+    include Yast::I18n
```

Comment:
I prefer to use ```Yast::``` in include because it makes everything more obvious to newcomers (and I see no drawback on it, it's actually even more robust).